cloudbuild/cloudbuild.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright © 2021 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+steps:
+- name: node
+  entrypoint: 'yarn'
+  args: ['install']
+- name: node
+  entrypoint: 'yarn'
+  args: ['run', 'bower-root']
+- name: node
+  entrypoint: 'yarn'
+  args: ['run', 'cdap-full-build']
```

Comment:
can you try `yarn run cdap-full-build-more-memory` instead if perhaps it would be faster?

Reply:
On the base instance, it didn't make much difference. I'm using it on the larger machine but I haven't switched back to see if it actually helps. I'll add this note to the optimization ticket.